### ISSUE TITLE
Dynamically List Available languages

### ISSUE THREAD

**Issue by bmol118** [aksoftware98/multilanguages#47]
I have an application with Blazor Server and was wondering if it is possible to get all of the available languages from the ILanguageContainerService?
This is more of just a nice to have as it reduces the amount of effort required when adding an additional language, but it is still easy enough to just add a new option when adding a language.

Specifically, I want to do something like:
```
  <InputSelect @bind-Value=SelectedLanguage >
	  @foreach(var registeredLanguage in languageContainer.RegisteredLanguages)
	  {
		  <option value=@registeredLanguage.CultureInfo>@registeredLanguage.Language</option>
	  }
  </InputSelect>
```

**Comment by aksoftware98** [comment]
Hi @bmol118 

Thank you so much for reaching out and using AKSoftware.Localization.MultiLanguages 

That's a very good idea, I will proceed with it to be released in v6.0 

Are you using the library currently?
If yes, can you provide some kind of review of it please, and what do you think about it? 

Thanks,
Ahmad

**Comment by bmol118** [comment]
@aksoftware98 
Yes I am currently using the library for an in development web application that will most likely be scaled to more than the current 2 languages it supports as I am at a global company.

So far it has been able to do everything I have wanted except for this feature.

Thanks!
